generators/app/templates/src/services/AuthService.ts

```diff
@@ -1,21 +1,23 @@
+import { Dispatch } from 'react';
 import AsyncStorage from '@react-native-community/async-storage';
 import api from '@config/api';
 import { actionCreators as authActions } from '@redux/auth/actions';
-
+import { CurrentUser } from '@interfaces/authInterfaces';
 // TODO: Adapt returned object to:
 //   sessionToken: usually currentUser.access_token
 //   renewId: usually currentUser.renew_id
 //   Also don't forget to add any relevant user data needed for your app.
-const formatUser = currentUser => currentUser;
+const formatUser = (currentUser: CurrentUser) => currentUser;
 
-export const setCurrentUser = currentUser => {
+export const setCurrentUser = (currentUser: CurrentUser) => {
   api.setHeader('Authorization', currentUser.sessionToken);
   return AsyncStorage.setItem('@Auth:currentUser', JSON.stringify(formatUser(currentUser)));
 };
-export const getCurrentUser = () => AsyncStorage.getItem('@Auth:currentUser').then(JSON.parse);
+export const getCurrentUser = () =>
+  AsyncStorage.getItem('@Auth:currentUser').then(value => JSON.parse(`${value}`));
```

Comment:
Is it okay the ``` `${value}` ```? Why?

Reply:
Because `JSON.parse` only accept a `string` an only that, so if you want to send it null it will not type. So all the `null` value will have to be converted to `null` string